src/main/app/src/App.tsx

```diff
@@ -1,33 +1,61 @@
-import React from 'react';
-import { HashRouter as Router, Switch, Route } from 'react-router-dom';
+import React, { ReactElement, useCallback, useState } from 'react';
+import { HashRouter as Router, Route, Switch } from 'react-router-dom';
 
 import LandscapeOverview from './Components/Landscape/Overview/Overview';
 import LandscapeMap from './Components/Landscape/Map/Map';
 import Man from './Components/Manual/Man';
 import Layout from './Components/Layout/Layout';
-import Events from './Components/Events/Events';
-import LandscapeDashboard from './Components/Landscape/Dashboard/Dashboard';
 import { Routes } from './interfaces';
 
-import './App.scss';
-import { ThemeProvider } from '@material-ui/core/styles';
-import theme from './Ressources/styling/theme';
-
 const App: React.FC = () => {
+  const [sidebarContent, setSidebarContent] = useState<ReactElement[]>([]);
+  const [pageTitle, setPageTitle] = useState<string>('');
+  const [findFunction, setFindFunction] = useState<Function>();
+
+  const ff = useCallback(
+    (args) => {
+      if (findFunction) findFunction(args);
+      else console.warn('find function not set');
+    },
+    [findFunction]
+  );
+
```

Comment:
Is `findFunction` a function for the search field? I would name it `searchFunction`

Reply:
It actually is "locateFunction" to scroll to the given item in the map. I will rename it.